tests/container_checker/test_container_checker.py

```diff
@@ -238,3 +238,40 @@ def test_container_checker(duthosts, enum_rand_one_per_hwsku_hostname, enum_rand
         # Wait for 70s to 80s  such that Monit has a chance to write alerting message into syslog.
         logger.info("Sleep '{}'s to wait for the alerting message...".format(sleep_time))
         time.sleep(sleep_time)
+
+
+def test_container_checker_telemetry(duthosts, rand_one_dut_hostname):
+    """Tests the feature of container checker.
+
+    This function will verify container checker for telemetry.
+
+    Args:
+        duthosts: list of DUTs.
+        rand_one_dut_hostname: Fixture returning dut hostname.
+
+    Returns:
+        None.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    container_name = "telemetry"
+
+    # Skip test if telemetry is disabled by other test
```

Comment:
>Skip test if telemetry is disabled by other test [](http://example.com/codeflow?start=6&length=48)

Is it a valid pre-situation? If it really happens, should we just reload config to ensure telemetry is not disabled? Skip may be not a good test strategy.

Reply:
test_container_checker can disable the telemetry container.
Now I restart telemetry service and continue the test.